docs/docs/getting-started/v3-migration.md

```diff
@@ -118,11 +120,13 @@ options: {
         major: {
           enabled: true
         },
-        fontStyle: function(context) {
-          return context.tick && context.tick.major ? 'bold' : undefined;
-        },
-        fontColor: function(context) {
-          return context.tick && context.tick.major ? '#FF0000' : undefined;
+        font: function(context) {
+          if (context.tick && context.tick.major) {
```

Comment:
Question on the implementation, does this also work?

```javascript
font: {
    fontStyle: (context) => context.tick && context.tick.major ? 'bold' : undefined
}
```

Reply:
No, that does not currently work.

Reply:
Do you think we need anything specific in the docs for that? It's different from other nested objects